crates/matrix-sdk/src/room/mod.rs

```diff
@@ -445,6 +445,21 @@ impl Room {
         )
         .await?;
 
+        // Save the loaded events into the event cache, if it's set up.
```

Comment:
Nice! Can you test this please? :)

Also consider adding a `RoomEventCache::save_events` method, so that we can save all the events at once without giving and releasing and giving and releasing etc. the lock.

Reply:
Done in https://github.com/matrix-org/matrix-rust-sdk/pull/3840/commits/919b5458fe886ec96cab4de9fe94b66bd50c404a, added some tests.